app/models/post.rb

```diff
@@ -1,2 +1,15 @@
 class Post < ApplicationRecord
+  belongs_to :user, foreign_key: 'author_id'
```

Comment:
- [ ] _Foreign key_ is improperly implemented. As per the given _ERD_, the _Post_ **association** should have _Post_ belong to _User_ **through** _Author_, an alias of _User_, like below:
`belongs_to :author, class_name: 'User'` 

    Kindly ensure to improve upon this aspect.